portal/src/locale-data/en.json

```diff
@@ -5,6 +5,7 @@
 
   "next": "Next",
   "save": "Save",
+  "order": "Order",
```

Comment:
The word order has many meanings. Let's rename the key to `DetailsListWithOrdering.order`.